extensions/mathjax/renderer.go

```diff
@@ -13,17 +14,37 @@ import (
 //go:embed js
 var js embed.FS
 
-const script = `<script async src="/js/mathjax.js"></script>`
+const script = `
+<script>
+MathJax = {
+  tex: {
+    displayMath: [['$$', '$$'], ['\\[', '\\]']],
+    inlineMath: [['$', '$'], ['\\(', '\\)']]
+  },
+  svg: {
+    fontCache: 'global'
+  }
+};
+</script>
+<script type="text/javascript" id="MathJax-script" async
+  src="/js/tex-chtml-full.js">
+</script>
+    `
 
 func init() {
 	RegisterStaticDir(js)
+	RegisterWidget(HEAD_WIDGET, 1, headScript)
 	RegisterBuildPage("/js/mathjax.js", false)
```

Comment:
after we used "/js/tex-chtml-full.js` does that mean we don't need `/js/mathjax.js` ? 

I tried to build a blog and it doesn't export the files in `js` because it's not registered,  I believe we'll need to Register all the first under `/js` to be exported on build

Another thing we can do is to implement a solution that adds `embed.FS` to the build process. maybe define `RegisterBuildStaticDir` that register a filesystem to be exported on build. this can be useful in the future for other extensions. 

For now I think you can walk the `embed.FS` and register all files with `RegisterBuildPage`

Reply:
i see. mh i don't really know that feature from go (`embed.FS`). I'll not have internet until next week, so if you get a solution in the meanwhile, would be awesome :) i can try then otherwhise

Reply:
Yeah I can fix it in the evening today